### DISCUSSION TITLE
How can we see the response object in the send request?

### DISCUSSION THREAD

**Discussion by Silenco** [postmanlabs/postman-flows#247] [Q&A]
I'm fairly new with Flow, I just want to make a request to my API and extract some values of the JSON file in the body... the problem I have is that I can't see the response in the output, anyone knows how can I see the schema in the outpur side, so I can use it in the For Each block?

![image](https://user-images.githubusercontent.com/6298344/153462774-97eebe58-9eb6-40a7-b7cc-deed43333a61.png)

**Comment by samay-v** [comment]
Hey @Silenco, to see the schema of the response body you'll need to add an example for the request in question. You can learn more [here ](https://learning.postman.com/docs/sending-requests/examples/)
